src/Basis/Constructors.jl

```diff
@@ -4,11 +4,28 @@
 
 using Polynomials
 
+# ------------------------------------------------------------------------------
+# conformal map functions for generating transformed polynomial bases
+# ------------------------------------------------------------------------------
+
 """
-Conformal mapping of Gauss ellipses to sausages using a truncated Taylor expansion of arcsin(s).
-See Figure 4.1 of Hale and Trefethen.
+```julia
+sausage(d)
+```
+# Arguments:
+- `d`: polynomial degree of truncated Taylor series expansion of arcsin(s).
+
+# Returns:
+Conformal mapping of Gauss ellipses to sausages using a truncated Taylor expansion of arcsin(s)
+See Figure 4.1 of Hale and Trefethen (2008).
+
+# Example:
+```jldoctest
+# sausage conformal map
+g, derivative(g) = LFAToolkit.sausage(9);
```

Comment:
Is this the right syntax?

Reply:
Uhm, derivative(g) should be written as g_prime. With this it will be consistent with the other two conformal maps